code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -315,8 +315,172 @@
 /datum/species/proc/handle_death(var/mob/living/carbon/human/H) //Handles any species-specific death events (such as dionaea nymph spawns).
 	return
 
-/datum/species/proc/handle_attack_hand(var/mob/living/carbon/human/H, var/mob/living/carbon/human/M) //Handles any species-specific attackhand events.
-	return
+/datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(attacker_style && attacker_style.help_act(user, target))//adminfu only...
+		return 1
+	if(target.health >= config.health_threshold_crit)
+		target.help_shake_act(user)
+		add_attack_logs(user, target, "Shaked")
```

Comment:
Please put this in `carbon/help_shake_act` and change it to `add_attack_logs(M, src, "Shaked", admin_notify = FALSE)`